packages/e2e/package.json

```diff
@@ -6,14 +6,14 @@
   "scripts": {
     "clean": "rimraf node_modules",
     "dev": "cypress open",
-    "test:examples": "cypress run --spec 'features/ui/hooks/*.feature','features/ui/components/**/*.feature'",
-    "test:authenticator": "cypress run --spec 'features/ui/components/authenticator/*.feature'",
-    "test:hooks": "cypress run --spec 'features/ui/hooks/*.feature'",
-    "test:geo": "cypress run --spec 'features/ui/components/geo/*.feature'",
-    "test:liveness": "cypress run --spec 'features/ui/components/liveness/*.feature'",
-    "test:storage": "cypress run --spec 'features/ui/components/storage/**/*.feature'",
-    "test:theme": "cypress run --spec 'features/ui/theme/*.feature'",
-    "test:guides": "cypress run --spec 'features/ui/guides/*.feature'",
+    "test:examples": "cypress run --spec 'features/ui/hooks/*.feature','features/ui/components/**/*.feature' --browser chrome",
+    "test:authenticator": "cypress run --spec 'features/ui/components/authenticator/*.feature' --browser chrome",
+    "test:hooks": "cypress run --spec 'features/ui/hooks/*.feature' --browser chrome",
+    "test:geo": "cypress run --spec 'features/ui/components/geo/*.feature' --browser chrome",
+    "test:liveness": "cypress run --spec 'features/ui/components/liveness/*.feature' --browser chrome",
```

Comment:
Definitely not a blocker for this PR but wondering if we should scope the `--browser` flag to just Liveness. Think ideally we should be able to test other browsers for the rest of the connected components

Reply:
Spoke with @reesscot @ioanabrooks and @0618 on this and we opted to switch to chrome for now since the default we use all around is  electron which is not indicative of our customer experience. Agree though we should probably be testing on more browswers in the future.

Also in this case during the e2e test run we only run `test:examples` command, these other ones are more for convenience but changed them to just match